test/async-dispose.js

```diff
@@ -0,0 +1,20 @@
+'use strict'
+
+const t = require('tap')
+const Fastify = require('../fastify')
+
+t.test('async dispose should close fastify', async t => {
```

Comment:
We should skip the tests for `<=16`, otherwise will throw as the API is not available there. At least until we drop support for `16` and `14`

Reply:
@metcoder95 How should I disable it?